cip.go

```diff
@@ -84,7 +94,7 @@ func main() {
 		os.Exit(0)
 	}
 
-	mfest, err := reg.ParseManifestFromFile(*manifestPtr)
+	mfest, err := reg.ParseManifestFromFile(*manifestPtr, *createManifestPtr)
```

Comment:
I don't understand why ParseManifestFromFile needs to care about creating manifests, which is already handled by CreateManifest().

Reply:
removed